Frameworks/AddressBook/ABRecord.mm

```diff
@@ -172,10 +170,9 @@ CFTypeRef ABRecordCopyValue(ABRecordRef record, ABPropertyID contactProperty) {
         return (__bridge_retained CFTypeRef)person.contact.notes;
     }
 
-    // The following properties either require the implementation of
-    // multi-values, or don't have a good 1-1 mapping between iOS contacts
+    // The following properties don't have a good 1-1 mapping between iOS contacts
     // and Windows contacts (alternate birthday, creation/modification dates,
-    // phonetic middle name).
+    // phonetic middle name, instant message/social profile).
```

Comment:
>  // phonetic middle name, instant message/social profile). [](start = 2, length = 59)

this comment should probably be expanded with a TODO to fill these in or an explanation stating they never will (if they never will we could also remove the cases).